static/app/views/performance/newTraceDetails/traceTree.tsx

```diff
@@ -249,6 +252,47 @@ export class TraceTree {
     return tree.build();
   }
 
+  static GetTraceType(root: TraceTreeNode<null>): TraceType {
+    const {transactions, orphan_errors} = root.children[0].value as TraceTree.Trace;
```

Comment:
You should use the type guard isTraceNode or type the input arg to be TraceTreeNode<TraceTree.Trace>. I would also add a test for this method